### PR TITLE
Pivot: Add icon to PivotItem

### PR DESCRIPTION
**Pull request by cschleiden**
#### Pull request checklist

- [x] Addresses an existing issue: #750
- [x] Include a change request file if publishing 
- [x] New feature
- [x] Documentation update

#### Description of changes

Add `linkIcon` property to `PivotItem` to allow showing an icon next to the pivot link.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx [hunk 1]
```diff
@@ -108,11 +108,22 @@ export class Pivot extends React.Component<IPivotProps, IPivotState> {
...
-    let countText;
+
+    let count;
     if (itemCount !== undefined && this.props.linkFormat !== PivotLinkFormat.tabs) {
-      countText = <span className='ms-Pivot-count'>({ itemCount })</span>;
+      count = <span className='ms-Pivot-count'>({ itemCount })</span>;
+    }
+
+    let icon: JSX.Element;
+    if (itemIcon !== undefined && this.props.linkFormat !== PivotLinkFormat.tabs) {
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx]: This seems weird to me; if you're in tabs mode the icons aren't displayed.

If we support icons we should probably just let the user render them.

Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx]: I just followed the existing code that doesn't allow `count` to be shown when in tabs mode, you think we should allow both?

File: common/changes/feature-pivotItemIcon_2017-01-05-23-29.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Add icon property to PivotItem",
```
Comment (mikewheaton) [inline common/changes/feature-pivotItemIcon_2017-01-05-23-29.json]: Change to "Pivot: Add icon to pivot item."

File: packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx (hunk 1 above)
Comment (mikewheaton) [inline packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx]: I'm not sure on the original reasons for hiding the count from tabs. Seems to me that regardless of tabs or links it should have the same functionality here.

File: common/changes/feature-pivotItemIcon_2017-01-05-23-29.json (hunk 2 above)
Comment (cschleiden) [inline common/changes/feature-pivotItemIcon_2017-01-05-23-29.json]: done

File: packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx (hunk 1 above)
Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx]: I've added both now, looks and works fine.

**Pull request merged**